vm/systemSmartContracts/governance.go

```diff
@@ -284,6 +284,10 @@ func (g *governanceContract) changeConfig(args *vmcommon.ContractCallInput) vmco
 
 // proposal creates a new proposal from passed arguments
 func (g *governanceContract) proposal(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if g.enableEpochsHandler.IsFlagEnabled(common.GovernanceDisableProposeFlag) && !g.enableEpochsHandler.IsFlagEnabled(common.GovernanceFixesFlag) {
```

Comment:
with these changes I don't think we still need the `Delegated` field in the new structure right?
it could be removed in the next PR.

Also we need to take care any view functions for the old proposals even if called after the new flag activation can work as expected.